docker/build/installers/install_gperftools.sh

```diff
@@ -37,7 +37,7 @@ download_if_not_cached "${PKG_NAME}" "${CHECKSUM}" "${DOWNLOAD_LINK}"
 tar xzf ${PKG_NAME}
 
 pushd "gperftools-gperftools-${VERSION}" >/dev/null
-    ./autogen.sh
+    ./autogen.sh || sleep 1 && ./autogen.sh
```

Comment:
why need sleep and try again?

Reply:
Don't know the exact reason but it works after sleep. `autogen.sh` fails when building the docker image. I started container out of partially built image, did `./autogen.sh` and it succeeds. so I tried `sleep 1` and the `./autogen.sh` succeeded and image is built.

Reply:
I don't have this problem, logically autogen.sh waits until the process ends

Reply:
I couldn't find the error it's throwing while building the image. Docker simply reports image build failed due to failing `autogen.sh`